explorer/client/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -172,6 +195,11 @@ function ItemView({ data }: { data: TxItemView }) {
                                     item.monotypeClass && styles.mono
                                 )}
                             >
+                                {links.length > 1 ? (
```

Comment:
You can also do `links.length > 1 && ()` to avoid needing a ternary here.

Reply:
the second clause of the ternary has a value so this doesn't seem to apply.